### DISCUSSION TITLE
admin password recovery

### DISCUSSION THREAD

**Discussion by t3chbg** [utmstack/UTMStack#713] [Q&A] [answered]
Good day, I just installed UTMstack using ISO image on VM and was not able to retrieve admin account password during installation and looking in /utmstack folder its not clear to me where to find the admin user password. I would appreciate if you could provide me with advice on how to retrieve admin account password thx

**Comment by agauttam** [comment, marked as answer]
Please use the command "sudo cat /root/utmstack.yml"

**Reply by c3s4rfred** [reply]
Hi, @t3chbg the command before, shows the admin web GUI password, if you are looking for the UTMStack web application credentials, check this link at the end of the page -> https://docs.utmstack.com/Installation/MasterServerInstallation
Best regards

**Reply by t3chbg** [reply]
Hi, @c3s4rfred, thank you but I still not able to find the password for the web application - The instructions suggest I look in /root/utmstack for passwords and other generated configs but as per my original post im not able to find generated password from installation in this location or its not clear to me where to find it in /utmstack

**Reply by c3s4rfred** [reply]
Check this image taken from the link that I sent before -> 
![image](https://github.com/utmstack/UTMStack/assets/17603499/ee94cb7e-edd9-4498-a692-f50d6b51df8d)

**Reply by c3s4rfred** [reply]
Please, open another discussion and provide as much information as you can, for example you could you access before?, what you changed after that?
Regards

**Reply by t3chbg** [reply]
I have reinstalled the entire server from scratch and Im still no closer to getting the login details to the web application as indicated in my original post
![image](https://github.com/utmstack/UTMStack/assets/60702255/37554bab-4e18-41b8-9648-1b7401c6f5ba)
